libs/blocks/marketo/marketo.js

```diff
@@ -32,6 +32,8 @@ export const formValidate = (form) => {
 };
 
 export const decorateURL = (destination, baseURL = window.location) => {
+  if (!(destination.startsWith('http') || destination.startsWith('/'))) return null;
+
   try {
     let destinationUrl = new URL(destination, baseURL.origin);
```

Comment:
Wouldn't this fail (rightfully) on L38 anyway, if the destination isn't a valid URL and then return null?

Reply:
No, if you look in the "before" link, you'll see that it encodes the string and turns it into a url: `https://main--milo--adobecom.hlx.page/Thank%20You.%20Your%20request%20has%20been%20submitted%20and%20you%20should%20receive%20an%20email%20shortly%20with%20instructions%20to%20log%20into%20Adobe%20Analytics`